frontend/test/__support__/server-mocks/index.ts

```diff
@@ -23,5 +25,6 @@ export * from "./setup";
 export * from "./store";
 export * from "./table";
 export * from "./timeline";
+export * from "./timeline";
```

Comment:
This is a duplicate from the line above.

```suggestion
```

Reply:
Removed